airflow/docker_test.go

```diff
@@ -290,6 +296,25 @@ func TestDockerGetContainerIDFailure(t *testing.T) {
 	assert.Contains(t, id, "")
 }
 
+func TestDockerWebserverHealthCheck(t *testing.T) {
+	composeMock, docker, _ := getComposeMocks()
+	mockCall := composeMock.On("Events", mock.Anything, mock.Anything, mock.Anything)
```

Comment:
Can't we use a more specific mock method than "Mock.Anything" ?

This is not a big deal, but we prefer to make mock assertions as specific as possible to avoid side effects in the tests.

Reply:
Given that we had to override the function logic to unit test our code and also since we were passing a function as an argument to the method, made it difficult to be more specific than `mock.Anything`. Hence I had skipped it for all the arguments initially, as of now have added specific arguments except the last one which takes in a function as an argument.